resources/styles/components/task/index.less

```diff
@@ -0,0 +1,35 @@
+.fixed-bar() {
```

Comment:
Can you prefix this with `task-` or include it in `mixins.less` (if it ends up being used elsewhere like the exercise builder)?